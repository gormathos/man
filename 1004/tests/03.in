3
-1.42726 -8.79644 -0.154967
5
6.92566 -2.72104 -4.26067 -1.31062 -2.30399
