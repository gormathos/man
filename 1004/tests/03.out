1.00000 10.37867 14.13914 1.94558
929.86648 20.16993 52.76525 -1.00905 12.23251
301.79168 -20.13022 -19.84108 -7.91266 -17.76044
